sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/Max.scala

```diff
@@ -57,3 +57,34 @@ case class Max(child: Expression) extends DeclarativeAggregate {
 
   override lazy val evaluateExpression: AttributeReference = max
 }
+
+abstract class AnyAggBase(arg: Expression)
```

Comment:
can we change this to something like `UnevaluableBooleanAggBase` and make also `EveryAgg` extend this, in order to avoid code duplication?

Reply:
@mgaido91 I had a confusion on where to house this class ? Is it okay if i just rename it and keep it in Max.scala ?

Reply:
maybe we can move it close to `UnevaluableAggrgate`. @cloud-fan @dilipbiswal WDYT?

Reply:
If it's hard to decide where to put it, I think putting it in a new file can be considered.